templates/logicmonitor/config.yml

```diff
@@ -0,0 +1,7 @@
+name: LogicMonitor
+description: SaaS-based performance monitoring for modern IT infrastructure
+version: latest
```

Comment:
Please, use concrete version instead implicit latest....Latest is no reproducible